### ISSUE TITLE
如何注册？

### ISSUE THREAD

**Issue by IMG-GAMES** [dhrdzy/DanMuGame#2]
下载后 环境已经安装 进入游戏注册页面 首次注册无人邀请 大善#2844 邀请码无效 无法注册

**Comment by dhrdzy** [comment]
不是  “大善#2844”，是  “大乖#2844”，昵称尽量用英文，平台端不用注册，直接点登录就行，帐号密码都不用输入，在需要的游戏内进行注册登录，平台端暂时就是个下载器
